app/AccountancyModule/Components/Cashbook/templates/ChitListControl.latte

```diff
@@ -45,67 +51,69 @@
+                                        target="_blank"
+                                        class="btn btn-sm btn-info d-none d-lg-inline"><i
+                                            class="fas fa-print"></i></a>
+                                <a class="ajax btn btn-sm btn-info d-none d-md-inline" n:href="chitScan:open! $chit->id" title="Skeny dokladů">
+                                    <i class="far fa-folder-open"></i>
+                                </a>
+                                <a n:if="$isEditable && ! $chit->locked" n:href="edit! $chit->id"
+                                        class="ajax btn btn-sm btn-primary ui--editChit"><i class="far fa-edit"></i></a>
 
-                            <a n:if="$isEditable && ! $chit->locked" n:href="remove! $chit->id"
-                                    onclick="return confirm('Opravdu chcete smazat paragon?')"
-                                    class="btn btn-danger btn-xs ui--removeChit"><i class="glyphicon glyphicon-remove"></i></a>
+                                <a n:if="$isEditable && ! $chit->locked" n:href="remove! $chit->id"
+                                        onclick="return confirm('Opravdu chcete smazat paragon?')"
+                                        class="btn btn-danger btn-sm ui--removeChit"><i class="far fa-trash-alt"></i></a>
 
-                            <a n:if="$isEditable && $chit->locked" n:href="showLocked" title="Paragon je uzamčen."
-                                    class="btn btn-warning btn-xs"><i class="glyphicon glyphicon-lock"></i></a>
+                                <a n:if="$isEditable && $chit->locked" n:href="showLocked" title="Paragon je uzamčen."
+                                        class="btn btn-warning btn-sm"><i class="fas fa-lock"></i></a>
 
-                            <a n:if="$isEditable && count(array_intersect($validInverseCashbookTypes, $chit->inverseCashbookTypes))"
-                                    class="ajax btn btn-xs btn-success hidden-xs hidden-sm" n:href="invertChitDialog:open! $chit->id"
-                                    title="Vytvořit protidoklad">
-                                <i class="fa fa-exchange"></i>
-                            </a>
+                                <a n:if="$isEditable && count(array_intersect($validInverseCashbookTypes, $chit->inverseCashbookTypes))"
+                                        class="ajax btn btn-sm btn-success d-none d-lg-inline" n:href="invertChitDialog:open! $chit->id"
+                                        title="Vytvořit protidoklad">
+                                    <i class="fas fa-exchange-alt"></i>
+                                </a>
+                            </div>
                         </td>
-                        <td{if $itemsCount > 1} rowspan="{$itemsCount+1}"{/if}>
-                            <span class="visible-xs visible-sm">{$chit->date|date:"j.n."}</span>
```

Comment:
*Neblokující* - Není to skrývání té ikony tady zbytečný?

Reply:
tady se to zkracuje o rok ne o ikonu

Reply:
tohle jsem tam zachovával historicky a taky jsem premyslel o tom co navrhujes, takze to tam rovnou přidám